templates/bot/ts/default/.webappignore

```diff
@@ -0,0 +1,17 @@
+.fx
```

Comment:
Do we also add this file in csharp bot template?

Reply:
I think in your last PR, you upload all build output but not all of the file in repository, so don't think we need this in csharp template